pkg/vex/vex.go

```diff
@@ -55,6 +56,18 @@ type VulnerabilityReference struct {
 
 func New() VEX {
 	now := time.Now()
+	// Support envvar for reproducible vexing
+	if d := os.Getenv("SOURCE_DATE_EPOCH"); d != "" {
+		sec, err := strconv.ParseInt(d, 10, 64)
+		if err == nil {
+			now = time.Unix(sec, 0)
+		} else {
+			t2, err := time.Parse(time.RFC3339, d)
+			if err == nil {
+				now = t2
+			}
+		}
+	}
```

Comment:
Question: This looks like basically the same implementation in both places, do you think it's worth factoring this out into a reusable function?

Reply:
OK, I've factored out the date reader to a function and reused it.